backend/app/services/executor_streaming/service.py

```
@@ -0,0 +1,599 @@
+                if result:
+                    subtask.result = result
+                subtask.status = SubtaskStatus.RUNNING
+                subtask.updated_at = datetime.now()
+                db.add(subtask)
+                db.commit()
+
+                self._last_db_save_times[subtask_id] = current_time
+                logger.debug(
+                    f"[ExecutorStreaming] Periodic DB save for subtask {subtask_id}"
+                )
+        except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)